cg_space/main.tf

```diff
@@ -3,46 +3,30 @@ data "cloudfoundry_org" "org" {
 }
 
 resource "cloudfoundry_space" "space" {
-  name = var.cf_space_name
-  org  = data.cloudfoundry_org.org.id
+  name      = var.cf_space_name
+  org       = data.cloudfoundry_org.org.id
+  allow_ssh = var.allow_ssh
 }
 
 ###
 # User roles
 ###
 
-data "cloudfoundry_user" "managers" {
-  for_each = var.managers
-  name     = each.key
-  org_id   = data.cloudfoundry_org.org.id
-}
-
-data "cloudfoundry_user" "developers" {
-  for_each = var.developers
-  name     = each.key
-  org_id   = data.cloudfoundry_org.org.id
-}
-
-data "cloudfoundry_user" "deployers" {
-  for_each = var.deployers
-  name     = each.key
-  org_id   = data.cloudfoundry_org.org.id
+locals {
+  manager_names   = setunion(var.managers, var.deployers)
```

Comment:
The reason both the old and new code declares a "deployer" to have both developer and manager roles is because manager is required in order to set the egress security group on the space.

Reply:
Right, but do you want the deployer to do that? It seems like this module should also manage [a `cloudfoundry_space_asgs` resource](https://registry.terraform.io/providers/cloudfoundry-community/cloudfoundry/latest/docs/resources/space_asgs) with the desired ASGs passed in a variable.

Reply:
Agreed, but the community asgs resource is totally broken and the official one exists only as a PR right now (though as soon as it's released I'll be adding it to the cg_space module so we can setup the space, user roles, and security groups in one go.